File: docs/components_page/components/badge/positioned.jl
using DashBootstrapComponents

badge = dbc_button(
    [
        "Notifications",
        dbc_badge(
            "99+",
            color = "danger",
            pill = true,
            text_color = "white",
            className = "position-absolute top-0 start-100 translate-middle",
        ),
    ],
    color = "primary",
    className = "position-relative",
)

File: docs/components_page/components/badge/positioned.R
library(dashBootstrapComponents)

badge <- dbcButton(
  list(
    "Notifications",
    dbcBadge(
      "99+",
      color = "danger",
      pill = TRUE,
      text_color = "white",
      className = "position-absolute top-0 start-100 translate-middle"
    )
  ),
  color = "primary",
  className = "position-relative"
)
